kernel/src/syscall_driver.rs

```diff
@@ -178,7 +179,7 @@ impl From<process::Error> for CommandReturn {
 /// kernel, and therefore there is no subscribe function for capsules to
 /// implement.
 #[allow(unused_variables)]
-pub trait Driver {
+pub trait SyscallDriver {
```

Comment:
All the other traits seem to be called `SystemCall...`, maybe this should be renamed to `SystemCallDriver`?

Reply:
Again, I didn't think there were

```
git grep SystemCall | wc -l
       0
git grep Syscall | wc -l
     288
```

Reply:
I was referring top the internal kernel functions like:
  - platform.rs:96